src/main/java/com/hubspot/jinjava/lib/tag/RawTag.java

```diff
@@ -41,7 +41,7 @@ public String interpret(TagNode tagNode, JinjavaInterpreter interpreter) {
     return result.toString();
   }
 
-  private String renderNodeRaw(Node n) {
+  public String renderNodeRaw(Node n) {
```

Comment:
There's a problem where Custom HTML tags are actually rendering jinja within RawTags`, so I'm updating the render call within `WidgetAttributeTag` to set the value to the raw tag string instead of what the raw tag renders to, that way it won't double render and strip out the jinja.